src/Search.js

```diff
@@ -1,46 +1,64 @@
-import React, { useState, useRef } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { Form, FormControl, Button, Container, Stack } from 'react-bootstrap/'
 import './Search.css';
 import { Link } from 'react-router-dom';
-import { getPokemon, getPokemonList } from './Frontend'
+import { Rating } from '@mui/material/';
+import { getPokemon } from './Frontend'
+import { getPokemonReviews } from './Backend'
 
 function Search() {
   const NameRef = useRef();
   const [Pokemon, setPokemon] = useState({});
-  const [PokemonList, setPokemonList] = useState([]);
   const [PokemonId, setPokemonId] = useState();
-  const [Offset, setOffset] = useState(0);
+  const [TotalReview, setTotalReview] = useState([]);
+  const [RatingAverage, setRatingAverage] = useState();
```

Comment:
Alright so this will keep track of the average rating of the pokemon when we load all of the scores in.

Reply:
Yup